examples/return_err/lib.rs

```diff
@@ -0,0 +1,74 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[ink::contract]
+mod return_err {
+
+    #[ink(storage)]
+    pub struct ReturnErr {
+        instantiated: bool,
```

Comment:
This field does not make sense. Because when you return `Result::Err(_)` the codegen should emit a call to `seal_return` with the revert flag set. Exactly the same behavior as for a message. So an `Err` implies `instantiated: false`.

Reply:
Well, this has nothing to do with `Err` and others. We enforce to have messages in the contract, so I just came up with this storage as demo